src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/AbsentReason.kt

```diff
@@ -11,6 +11,12 @@ enum class AbsentReason(val label: String) {
   Refused("Refused to attend"),
   RefusedIncentiveLevelWarning("Refused to attend");
 
+  val labelWithWarning: String
+    get() = if (iepTriggers.contains(this)) "$label - incentive level warning added" else label
+
+  val labelWithShortWarning: String
+    get() = if (iepTriggers.contains(this)) "$label with warning" else label
```

Comment:
for stats we only want a short warning - not used yet, but will be in subsequent PRs